pkg/operator/ceph/cluster/mon/drain.go

```diff
@@ -97,3 +119,19 @@ func (c *Cluster) allowMonDrain(request types.NamespacedName) error {
 	}
 	return nil
 }
+
+func UsePDBV1Version(Clientset kubernetes.Interface) (bool, error) {
+	k8sVersion, err := k8sutil.GetK8SVersion(Clientset)
+	if err != nil {
+		return false, errors.Wrap(err, "failed to fetch get k8s version")
+	}
+	logger.Debug("k8ver", k8sVersion)
+	// minimum k8s version required for v1 PodDisruptionBudget is 'v1.21.0'. Apply v1 if k8s version is at least 'v1.21.0', else apply v1beta1 PodDisruptionBudgetJob.
+	minVersionForPDBV1 := "1.21.0"
+	usePDBV1 := k8sVersion.AtLeast(version.MustParseSemantic(minVersionForPDBV1))
+	if usePDBV1 {
+		return true, nil
+	} else {
+		return false, nil
+	}
```

Comment:
no need of else if returning inside `if`
```suggestion
	if usePDBV1 {
		return true, nil
	} 
	return false, nil
	
```